spec/ruby/library/socket/tcpsocket/shared/new.rb

```diff
@@ -14,14 +14,22 @@
     }
   end
 
-  ruby_version_is "3.0" do
+  ruby_version_is "3.0"..."3.1" do
```

Comment:
Should have been `..."3.2"`, fixed in ruby/spec@c3677cf497b864cd731d175fe8a72fbf53fc882f

Reply:
Thanks!